README.md

```diff
@@ -54,6 +54,9 @@ The 4 areas of the box model are:
 
 5. Explain why git is valuable to a team of developers.
 
+In a nutshell, Git, being a Version Control System, allows you to collaborate on large projects without running the risk of conflicting or site-breaking changes. It also allows you to have multiple sequential development processes running simultaneously, keep track of any and all changes, and to revert to a previous version should the need arise.
+Git makes it easy to split tasks among a team, each working on their own little section or piece, and review and further improve them individually without them affecting the 'master' version of the software. 
+
 You are expected to be able to answer all these questions. Your responses contribute to your Sprint Challenge grade. Skipping this section *will* prevent you from passing this challenge.
 
 ## Project Set Up
```

Comment:
Good job on explaining your answers in detail while giving examples to your questions.